docs/cloud/README.md

```diff
@@ -1,5 +1,23 @@
+| --- | --- | --- |
+| [AWS](AWS) | [Intel/AMD (x86_64)](https://aws.amazon.com/marketplace/pp/prodview-mku4y3g4sjrye)<br>[ARM64 (Aarch64)](https://aws.amazon.com/marketplace/pp/prodview-zgsymdwitnxmm) | [Intel/AMD (x86_64)](https://aws.amazon.com/marketplace/pp/prodview-ykmb6re2rcouy)<br>[ARM64 (Aarch64)](https://aws.amazon.com/marketplace/pp/prodview-pvttztrj2uzjk) | 
+| [Azure](Azure) | [Intel/AMD (x86_64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64?tab=Overview)<br>[ARM64 (Aarch64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm?tab=Overview)<br>[HPC](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-hpc?tab=Overview) | [Intel/AMD (x86_64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64?tab=Overview)<br>[ARM64 (Aarch64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm?tab=Overview) |
+| [Generic Cloud (Cloud-init)](Generic-cloud)| [Intel/AMD (x86_64)](https://repo.almalinux.org/almalinux/8/cloud/x86_64/images/)<br>[ARM64 (Aarch64)](https://repo.almalinux.org/almalinux/8/cloud/aarch64/images)<br>[IBM Z (s390x)](https://repo.almalinux.org/almalinux/8/cloud/s390x/images/)<br>[Power LE (ppc64le)](http://repo.almalinux.org/almalinux/8/cloud/ppc64le/images/) | [Intel/AMD (x86_64)](https://repo.almalinux.org/almalinux/9/cloud/x86_64/images/)<br>[ARM64 (Aarch64)](https://repo.almalinux.org/almalinux/9/cloud/aarch64/images/)<br>[IBM Z (s390x)](https://repo.almalinux.org/almalinux/9/cloud/s390x/images/)<br>[Power LE (ppc64le)](https://repo.almalinux.org/almalinux/9/cloud/ppc64le/images/) |
+| [Google Cloud](Google) | [Intel/AMD (x86_64)](https://console.cloud.google.com/marketplace/product/almalinux-cloud/almalinux-8)| [Intel/AMD (x86_64)](https://console.cloud.google.com/marketplace/product/almalinux-cloud/almalinux-9) |
+| [OpenNebula](OpenNebula) | [Intel/AMD (x86_64)](https://repo.almalinux.org/almalinux/8/cloud/x86_64/images/)<br>[ARM64 (Aarch64)](https://repo.almalinux.org/almalinux/8/cloud/aarch64/images) |[Intel/AMD (x86_64)](https://repo.almalinux.org/almalinux/9/cloud/x86_64/images/)<br>[ARM64 (Aarch64)](https://repo.almalinux.org/almalinux/9/cloud/aarch64/images/)|
+| [Oracle Cloud Infrastructure](OCI) | [Intel/AMD (x86_64)](https://cloudmarketplace.oracle.com/marketplace/en_US/listing/125544666)<br>[ARM64 (Aarch64)](https://cloudmarketplace.oracle.com/marketplace/en_US/listing/125567282) |[Intel/AMD (x86_64)](https://cloudmarketplace.oracle.com/marketplace/en_US/listing/127985411)<br>[ARM64 (Aarch64)](https://cloudmarketplace.oracle.com/marketplace/en_US/listing/127985893) |
+
+Additionally, for those wishing to simulate cloud environments or test locally, we also provide options to run the Cloud image on your local machine. Dive in and experience the pinnacle of cloud-based Linux distributions with AlmaLinux.
```

Comment:
Could you mention this guide on wiki please?

https://wiki.almalinux.org/cloud/Generic-cloud-on-local.html